src/lua/compat.c

```diff
@@ -5,8 +5,40 @@
  */
 
 #include <lua/compat.h>
+#include "serializer.h"
+
+int
+lbox_msgpuck_escape_forward_slash_toggle(struct lua_State *L)
+{
+	assert(lua_isboolean(L, -1) && "boolean argument expected");
+	bool esc_slash = lua_toboolean(L, -1);
+	lua_pop(L, 1);
```

Comment:
Minor: there is no need to explicitly pop this argument from Lua stack; `return 0;` below signals Lua VM to do it later.

Reply:
Fixed.